### PR TITLE
feat: fixes #220

### LINKED ISSUE #220: improve Subscribe Mailing-list more clear
Hi, to encourage more users to subscribe to our mailing list, could we make this section more clear on the Homepage? even in the navbar or footer?

We only have one subscribe guide currently.

[1] http://apisix.apache.org/docs/general/subscribe-guide

### PR DESCRIPTION
**Pull request by 1502shivam-singh**
Fixes: #220

Changes:

View discussion and changes here with screenshots 
https://github.com/apache/apisix-website/issues/220#issuecomment-789746929

### CODE REVIEW

File: website/src/css/customTheme.css [hunk 1]
```diff
@@ -501,3 +502,40 @@ a:hover {
...
+}
+
+.news-logo-svg{
+  height: 42px;
+}
+
+.news-text{
+  color: var(--color-primary);
+  text-align: center;
+}
```
Comment (juzhiyuan) [inline website/src/css/customTheme.css]: Hi, please add a blank line here, others LGTM.

File: website/src/css/customTheme.css [hunk 2]
```diff
@@ -87,7 +87,7 @@ header h2 {
 }
 @media only screen and (min-device-width: 375px) and (max-device-width: 736px) {
   .home-splash .title {
-    font-size: 3.52rem;
+    font-size: 3.23rem;
```
Comment (qier222) [inline website/src/css/customTheme.css]: Why change this?

File: website/src/css/customTheme.css (hunk 1 above)
Comment (1502shivam-singh) [reply website/src/css/customTheme.css]: An extra line at the end right, line 542.

File: website/src/css/customTheme.css (hunk 2 above)
Comment (1502shivam-singh) [reply website/src/css/customTheme.css]: > Why change this?

commit - "Fixed the size of hero slogan text, keyword overflowed earlier"

When "Performant" came up the keyword broke to a newline, leaving a single "A" in one line. To keep the text together, I decreased the font size.

Comment (1502shivam-singh) [reply website/src/css/customTheme.css]: Talking about this - 

image

Also the CTAs get cropped out of view.

File: website/src/css/customTheme.css (hunk 1 above)
Comment (juzhiyuan) [reply website/src/css/customTheme.css]: image

File: website/src/css/customTheme.css (hunk 2 above)
Comment (qier222) [reply website/src/css/customTheme.css]: I've fixed the problem, but haven't pushed the code yet

Comment (1502shivam-singh) [reply website/src/css/customTheme.css]: Okay, is your solution same? If not, then maybe just push it in your next PR, I guess.

Comment (qier222) [reply website/src/css/customTheme.css]: Not the same, check out this.
https://github.com/apache/apisix-website/pull/241/commits/c39ee9ed13485fa604dea5a253592c283c5e4e09

Comment (1502shivam-singh) [reply website/src/css/customTheme.css]: Should I make these changes in this push, if you don't mind.

Comment (qier222) [reply website/src/css/customTheme.css]: Please don't shrink down the font size, my purpose is to make the title fit the screen, make it looks less "left-aligned".

Comment (1502shivam-singh) [reply website/src/css/customTheme.css]: Okay, then can you push your changes in your next pull request.

Comment (qier222) [reply website/src/css/customTheme.css]: https://github.com/apache/apisix-website/pull/241/commits

Comment (1502shivam-singh) [reply website/src/css/customTheme.css]: Got it. Will make those CSS changes 👍

Comment (1502shivam-singh) [reply website/src/css/customTheme.css]: @qier222 you are fine with this, **or is it work in progress.** 
image

Can't really check on a real iphone, as I use android.

Comment (qier222) [reply website/src/css/customTheme.css]: Looks good on my build. Try clear your browser cache?
image

Comment (1502shivam-singh) [reply website/src/css/customTheme.css]: `.home-splash .title {
  font-size: 5rem;
  font-weight: 800;
  color: #101827;
}`

`
@media only screen and (min-device-width: 376px) and (max-device-width: 736px) {
  .home-splash .title {
    font-size: 3.52rem;
  }
}
`

`
@media only screen and (min-device-width: 320px) and (max-width: 375px) {
  .home-splash .title {
    font-size: 3rem;
  }
}
`

No, I mean technically 375px is in neither of the media queries, so first style is applied, am I right. But, if you feel it looks fine on your system and phone, I'm fine then.

Comment (1502shivam-singh) [reply website/src/css/customTheme.css]: Is it? Maybe I am having an issue that its not showing the recent load, or something 😕. Just cleared the cache too, I am using the Brave browser. Please check the main website again maybe.

File: website/src/pages/index.js [hunk 3]
```diff
@@ -134,7 +134,7 @@ const Showcase = () => {
         <p>
           This project is used by all these folks
           <br />
-          Are you using this project?
+          Are you using this project?&nbsp;
           <a
             href="https://github.com/apache/apisix/blob/master/docs/en/latest/powered-by.md"
```
Comment (juzhiyuan) [inline website/src/pages/index.js:139]: Hi, would you mind fixing this link in this PR?

The correct one is `https://github.com/apache/apisix/blob/master/powered-by.md`

**Review by juzhiyuan — APPROVED** [review]
Others LGTM

**Review by LiteSun — APPROVED** [review]

File: website/src/pages/index.js (hunk 3 above)
Comment (1502shivam-singh) [reply website/src/pages/index.js:139]: Okay, will fix this issue, in #250

**Pull request merged**